city/routers.py

```diff
@@ -0,0 +1,55 @@
+from fastapi import APIRouter, HTTPException
+
+from city.models import City
+from city import schemas, crud
+from dependencies import CommonSession
+from sqlalchemy import select
+
+
+router = APIRouter()
+
+
+@router.get("/cities", response_model=list[schemas.CityList])
```

Comment:
```suggestion
@router.get("/cities/", response_model=list[schemas.CityList])
```

don't forget to add slashes